.github/scripts/check-grpc-coverage.py

```
@@ -0,0 +1,326 @@
+    total_implemented = len([q for q, info in known_queries.items() if info.get("status") == "implemented"])
+    total_excluded = len([q for q, info in known_queries.items() if info.get("status") == "excluded"])
+    
+    report_lines.append(f"\nTotal known queries: {len(known_queries)}")
+    report_lines.append(f"  - Implemented: {total_implemented}")
+    report_lines.append(f"  - Not implemented: {total_not_implemented}")
+    report_lines.append(f"  - Excluded: {total_excluded}")
+    
+    # List all not implemented queries
+    not_implemented_queries = [q for q, info in known_queries.items() if info.get("status") == "not_implemented"]
+    if not_implemented_queries:
+        report_lines.append(f"\nNot implemented queries:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)